package.json

```diff
@@ -14,7 +14,7 @@
     "e2e:dev": "./e2e/start-and-run-suite dev",
     "jest": "jest --notify --watch",
     "jest-ci": "mkdir -p reports/junit && export JEST_JUNIT_OUTPUT_DIR=reports/junit && jest --ci --reporters=default --reporters=jest-junit --maxWorkers 2",
-    "lint": "eslint public/app e2e/suite1 public/test --ext=.js,.ts,.tsx",
+    "lint": "eslint . --ext .js,.tsx,.ts --cache",
```

Comment:
I guess `packages:lint` script can now be removed too

Reply:
Yep. No need for that.